sympy/sets/tests/test_sets.py

```diff
@@ -941,3 +942,13 @@ def test_issue_10113():
 def test_issue_10248():
     assert list(Intersection(S.Reals, FiniteSet(x))) == [
         And(x < oo, x > -oo)]
+
+
+def test_set_in_interval():
+    x = Symbol('x', real=True)
+    assert x in Interval(x, x+4)
+    assert Interval(1, 2) not in Interval(0, 5)
```

Comment:
This sure seems counterintuitive -- so we can have an intersection of two intervals but still say that the smaller one is not contained in the bigger one? If an integer can be in an Interval, what's wrong with seeing if a set (FiniteSet) of integers is in the Interval...or *all* numbers in a range (Interval) is in the Interval?